flutter_vlc_player/README.md

```diff
@@ -176,12 +208,23 @@ If you have some changes made to the iOS app, recreate the app using above metho
 
 Be sure to follow instructions above after 
 
-<hr>
+<br>
 
 ### Breaking Changes (from V4 to V5)
 Entire platform has been refactored in v5. It will require a refactor of your app to follow v5. 
 
+<br>
 <hr>
+<br>
+
+
+## Known Issues
+<b>1)</b> there is an issue in the <u>recording feature of underlying vlc library (not the Dart wrapper we're supporting)</u>, that if you do not call the `stopRecording()` method after `startRecording(...)` method and the video reaches to end the recorded video path is not getting updated, because the vlc stop recording event never gets fired.
```

Comment:
Let's rephrase a bit:

```suggestion
<b>1)</b> The video recording feature is problematic in iOS: if the video reaches its end while you're recording it, the underlying `vlckit` library fails to finalise the recording process, and we cannot retrieve the recorded file. The issue is reported and tracked here: [https://code.videolan.org/videolan/VLCKit/-/issues/394](https://code.videolan.org/videolan/VLCKit/-/issues/394) (see last comment from September 22, 2020)
```

Reply:
The issue is prevalent on Android too. I was able to reproduce on Android.

Reply:
Thanks for pointing that out! 
Curiously, it didn't reproduce on my device -- maybe the problem is device-specific.

In any case, I think it would be good to leave a link for the tracked issue on VLC Gitlab. Given that the problem exists on both platforms, I suspect that the bug should be in `libvlc` per se, not its iOS/Android wrappers. Could you please try finding the issuetracker on their Gitlab? 
If not, I'll be able to make a quick search on Monday.

Reply:
i updated readme and i also searched about this issue on libvlc-android. 
I only found the following link:
[https://code.videolan.org/videolan/vlc-android/-/issues/2117](https://code.videolan.org/videolan/vlc-android/-/issues/2117)

Reply:
My thought was too look for similar issues on the Gitlab repo of the core `libvlc` library: https://code.videolan.org/videolan/vlc/-/issues.

Reply:
I looked at that repo too, but i didn't find any issue.